lib/prometheus_exporter/server/collector.rb

```diff
@@ -78,6 +78,7 @@ def register_metric(metric)
     def register_metric_unsafe(obj)
       name = obj["name"]
       help = obj["help"]
+      opts = symbolize_keys(obj["opts"] || {})
```

Comment:
Do we need symbolize keys here? If so why, also why run a bunch of code if not opts were given?

Reply:
I think we were symbolizing keys because that's what the summary and histogram classes are looking for: https://github.com/discourse/prometheus_exporter/blob/master/lib/prometheus_exporter/metric/histogram.rb#L10. If people are using those models we would break them by changing the options hash to use string keys instead of symbols. So it seems better to continue symbolizing the keys.

This could be moved down to only happen in the summary and histogram "when" blocks but it seems like de-duping code is more advantagious here since running inject on a blank hash shouldn't really do anything.